### PR TITLE
Task/flwagner/consolidated

### PR DESCRIPTION
**Pull request by h2floh**
Slight modifications of chart and project resources after debugging POI Skill. However not directly related to debug of POI Skill.

### CODE REVIEW

File: aks/charts/letsencrypt/values.yaml [hunk 1]
```diff
@@ -4,44 +4,25 @@
...
-  annotations:
-    kubernetes.io/ingress.class: traefik
-  rules:
-    endpoint: 
-      host: vameetsaks.koreacentral.cloudapp.azure.com
-      paths:
-      - path: /
-        serviceName: aadlogin
-        servicePort: 443
+
```
Comment (angie4u) [inline aks/charts/letsencrypt/values.yaml]: This is a question! Do we inject those values (for example, aks host url, IP address etc.) from outside of the values.yaml?

**Comment by angie4u** [comment]
@h2floh  Thank you so much for the update! I think it is a good idea to create a new directory for letsencrypt! It makes this code more easy to read!

File: aks/charts/letsencrypt/values.yaml (hunk 1 above)
Comment (h2floh) [reply aks/charts/letsencrypt/values.yaml]: We don't use the ingress controller therefore I removed all options except disabled. Maybe we should remove the ingress template and all ingress related variables?!

Comment (angie4u) [reply aks/charts/letsencrypt/values.yaml]: Oh that sounds good! Could you update those parts as well? 😁

Comment (h2floh) [reply aks/charts/letsencrypt/values.yaml]: I already deleted the ingress template, removed the associated values from values.yaml, and also deleted unnecessary yaml files.

**Review by angie4u — APPROVED** [review]
Thanks for your work! 😎

**Pull request merged**